app/javascript/controllers/copy_controller.js

```diff
@@ -0,0 +1,31 @@
+import { Controller } from "@hotwired/stimulus"
+import ClipboardJS  from "clipboard"
+window.ClipboardJS = ClipboardJS
```

Comment:
Is this really necessary or just for debug purposes? Maybe this can be moved to the `application.js` instead

Reply:
Ops, my bad this was just for debug. Going to remove it